notebooks/MAST/HSC/CASJOBS/HCV_CASJOBS/HCV_casjobs_demo.ipynb

```diff
@@ -0,0 +1,995 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_hla_cutout(imagename, ra, dec, size=33, autoscale=99.5, asinh=True, zoom=1):\n",
+    "    \n",
+    "    \"\"\"Get JPEG cutout for an image\"\"\"\n",
+    "    \n",
+    "    url = \"https://hla.stsci.edu/cgi-bin/fitscut.cgi\"\n",
+    "    r = requests.get(url, params=dict(ra=ra, dec=dec, size=size, format=\"jpeg\",\n",
+    "                                      red=imagename, autoscale=autoscale, asinh=asinh, zoom=zoom))\n",
+    "    im = Image.open(BytesIO(r.content))\n",
+    "    return im\n",
+    "\n",
+    "\n",
+    "# sort images by magnitude from brightest to faintest\n",
+    "phot = nova_606\n",
+    "isort = np.argsort(phot['CorrMag'])\n",
+    "# select the brightest, median and faintest magnitudes\n",
+    "ind = [isort[0], isort[len(isort)//2], isort[-1]]\n",
+    "\n",
+    "# we plot zoomed-in and zoomed-out views side-by-side for each selected image\n",
+    "nim = len(ind)*2\n",
+    "ncols = 2 # images per row\n",
+    "nrows = (nim+ncols-1)//ncols\n",
+    "\n",
+    "imsize1 = 19\n",
+    "imsize2 = 101\n",
+    "mra = nova_tab['RA'][0]\n",
+    "mdec = nova_tab['Dec'][0]\n",
+    "\n",
+    "plt.rcParams.update({\"font.size\": 16})\n",
+    "plt.figure(1, (12, (12/ncols)*nrows))\n",
+    "t0 = time.time()\n",
+    "ip = 0\n",
+    "for k in ind:\n",
+    "    im1 = get_hla_cutout(phot['ImageName'][k], mra, mdec, size=imsize1)\n",
+    "    ip += 1\n",
+    "    plt.subplot(nrows, ncols, ip)\n",
+    "    plt.imshow(im1, origin=\"upper\", cmap=\"gray\")\n",
+    "    plt.title(f\"{phot['ImageName'][k]} m={phot['CorrMag'][k]:.3f}\", fontsize=14)\n",
+    "    im2 = get_hla_cutout(phot['ImageName'][k], mra, mdec, size=imsize2)\n",
+    "    ip += 1\n",
+    "    plt.subplot(nrows, ncols, ip)\n",
+    "    plt.imshow(im2, origin=\"upper\", cmap=\"gray\")\n",
+    "    xbox = np.array([-1, 1])*imsize1/2 + (imsize2-1)//2\n",
+    "    plt.plot(xbox[[0, 1, 1, 0, 0]], xbox[[0, 0, 1, 1, 0]], 'r-', linewidth=1)\n",
+    "    plt.title(f\"{phot['ImageName'][k]} m={phot['CorrMag'][k]:.3f}\", fontsize=14)\n",
+    "plt.tight_layout()\n",
+    "print(f\"{(time.time()-t0):.1f} s: got {ip} cutouts\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compare the HCV automatic classification to expert validations <a name=\"expert\"></a>\n",
+    "\n",
+    "The HCV includes an automatic classification `AutoClass` for candidate variables as well as an expert validation for some fields that were selected for visual examination.  For this example, we select all the objects in the HCV that have expert classification information."
```

Comment:
Who are the "experts"?  I'm sure this is documented somewhere, but it would be good to included at least a link or something to explain a bit more about what this means.

Reply:
@rlwastro can you add details for this?

Reply:
Yup, I will get this one.

Reply:
I added links both to the paper and to an earlier documentation block in the notebook that gives more detail on this.  (There are already many links to the paper, but one more doesn't hurt.)  This is in a pull request to @trenton3983's fork.